flixel/input/gamepad/FlxGamepadManager.hx

```diff
@@ -303,6 +319,8 @@ class FlxGamepadManager implements IFlxInputManager
 	@:allow(flixel.FlxG)
 	private function new() 
 	{
+		deviceConnected = new FlxTypedSignal<FlxGamepad->Void>();
```

Comment:
I wonder if these abstract ```@:multiType``` signals actually generate fast C++ code. Usually generic code must be marked with ```@:generic``` and you have to use classes that perform the signal action in order to get the benefits from generics.